### PR TITLE
sar-5 Added ShopifyOrderFulfillment API calls, LocalDateTimeSerializer into ShopifyCustomer, ShopifyOrder, and ShopifyOrderFulfillment

### PR DESCRIPTION
**Pull request by abhisheksangani**

### CODE REVIEW

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentRequest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```
Comment (dftrakesh1) [inline src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentRequest.java]: I think this annotation is not needed.

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.dft.api.shopify.v202307.model.fullfillment;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (dftrakesh1) [inline src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java]: I think this annotation is not needed.

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (dftrakesh1) [inline src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java]: I think this annotation is not needed.

**Review by dftrakesh1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentRequest.java (hunk 1 above)
Comment (abhisheksangani) [reply src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentRequest.java]: Removed all unnecessary annotation

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java (hunk 2 above)
Comment (abhisheksangani) [reply src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java]: Removed this annotation.

File: src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java (hunk 3 above)
Comment (abhisheksangani) [reply src/main/java/com/dft/api/shopify/v202307/model/fullfillment/ShopifyFulfillmentWrapper.java]: Removed this annotation.

**Pull request merged**